otherlibs/chrome-trace/src/chrome_trace.mli

```diff
@@ -87,6 +87,13 @@ module Event : sig
     ?tdur:Timestamp.t -> ?args:args -> dur:Timestamp.t -> common_fields -> t
 
   val to_json : t -> Json.t
+
+  type scope =
```

Comment:
I don't fully understand what you're doing this but why do we need this type if the only value you use is `Global`?

Reply:
This library is meant to be general purpose (although still in incubation phase) so the various scopes that I've defined all come from the standard for this trace format.

Reply:
It's worth saying that in a comment.

Reply:
What should I mention? It already says the library is unstable and is subject to change. I think that's basically the same as saying it's incubation.

Reply:
I meant, say that the scopes "all come from the standard for this trace format".